index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+
+<head>
+  <meta charset="utf-8">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <title>Jornada Trybe</title>
+</head>
+
+<body>
+  <!--2. Adicione uma barra superior com um título-->
+  <header id="cabecalho">
+    <h1 id="titulo">Jornada Trybe</h1>
+    <nav>
+      <ul>
+        <li>Aprendizado na Trybe</li>
+        <li>Contatos</li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <!--3. Adicione uma foto sua à página  -->
+    <div class="imagem listas">
+      <img id="minha_foto" src="minha_foto.jpeg" alt="minha_foto">
+    </div>
+    <!--4. Adicione uma lista de lições aprendidas à página-->
+    <section class="listas">
+      <h2>Liçoes Aprendidas</h2>
```

Comment:
@GabrielFMPinheiro acredito que tenha um erro tipográfico neste item.

```suggestion
      <h2>Lições Aprendidas</h2>
```

Reply:
Muito obrigado Israel! Acabei me preocupando muito com o código e nem revisei os meus textos. Irei acertar!